code.gs

```diff
@@ -32,14 +32,14 @@ var myEmail = 'insertyouremailhere@someemail.com';
  * If you leave it blank it will default to your Google account name.
  * Note: this may not work when using a Gmail address sending emails to itself.
  */
-var emailSenderName = 'Birthday Notifications';
+var emailSenderName = 'Contacts Events Notifications';
 
 /*
- * ID OF THE BIRTHDAY CALENDAR
+ * ID OF THE CONTACTS EVENTS CALENDAR
  *
- * Open up https://calendar.google.com, in the menu on the left click on the arrow next to the birthday calendar
```

Comment:
Does this calendar have a default name?

In italian it's called "Compleanni ed eventi" which would translate into "Birthdays and events". I don't know the exact english version, but we should use it here.